src/ast/rewriter/seq_rewriter.cpp

```diff
@@ -329,7 +329,13 @@ br_status seq_rewriter::mk_app_core(func_decl * f, unsigned num_args, expr * con
     switch(f->get_decl_kind()) {
 
     case OP_SEQ_UNIT:
-        return BR_FAILED;
+        // TODO configuration param
+        if (true) {
+            SASSERT(num_args == 1);
+            return mk_seq_unit(args[0], result);
```

Comment:
This does not need to be a configuration parameter. Just return mk_seq_unit(args[0], result)